packages/nimble-blazor/Tests/NimbleBlazor.Tests/Unit/Components/NimbleThemeProviderTests.cs

```diff
@@ -47,7 +48,28 @@ public void NimbleThemeProvider_DirectionIsSet(Direction value, string expectedA
     {
         var themeProvider = RenderNimbleThemeProvider(value);
 
-        Assert.Contains(expectedAttribute, themeProvider.Markup);
+        var expectedMarkup = $"direction=\"{expectedAttribute}\"";
+        Assert.Contains(expectedMarkup, themeProvider.Markup);
+    }
+
+    [Theory]
+    [InlineData(null)]
+    [InlineData("en-US")]
+    [InlineData("de-DE")]
+    [InlineData("123")]
+    public async void NimbleThemeProvider_LangIsSet(string value)
+    {
+        var themeProvider = RenderNimbleThemeProvider(value);
+
+        if (value == null)
```

Comment:
We've been trying to avoid branching within our test code, so it may be better to split the `null` case out into its own test.